libs/spark/README.md

```diff
@@ -0,0 +1,66 @@
+# spark
+
+This library was generated with [Nx](https://nx.dev).
+
+## Running unit tests
+
+Run `nx test spark` to execute the unit tests via [Jest](https://jestjs.io).
+
+## Prenda Spark
+
+- [Documentation](#documentation)
+- [Development](#development)
+- [Testing](#testing)
+- [WIP and TODOS](#wip-and-todos)
+- [Install](#install)
+- [Usage](#usage)
+
+---
+
+## Documentation
+
+> Official Docs: https://prenda-school.github.io/prenda-documentation/
+
+Prenda Spark extends Material UI react components (https://material-ui.com/) by overriding select styles to match the Prenda Design System philosophy
+
+## Development
+
+1. Clone the repository.
+2. Ensure you have Node Version Manager installed on your system ([see instructions](https://github.com/nvm-sh/nvm#installing-and-updating)).
+3. Run `> nvm use` to ensure specified node version is used.
+4. Run `> npm ci` to install dependencies.
+5. Run `> nx run spark:storybook` to trigger nx to run the spark project in develop mode (open `http://localhost:4200/` in your default browser to view.)
+
+Now you can develop components within the `libs/spark/src/` directory and see changes update in realtime.
+
+## Testing
+
+Run tests with `nx test spark` for tests using `@testing-library/react`(Prenda Spark uses Jest tests with React Testing Library) . To run end-to-end tests with [Cypress.io](https://www.cypress.io/), run `nx run spark-e2e`, and add the `--watch` flag to run using the Cypress test runner.
```

Comment:
It might be better to use `nx run spark:test` here, or to use the infix style notation `nx test spark` everywhere. As long as they are all consistent either one is fine.